src/lib/i18n.tsx

```diff
@@ -95,7 +95,9 @@ interface ProviderProps {
 export function Provider({ locale, forceRenderAfterLocaleChange = true, onActivate, children }: ProviderProps) {
   useEffect(() => {
     dynamicActivate(locale)
-      .then(() => onActivate?.(locale))
+      .then(() => {
```

Comment:
This shouldn't have been changed?

Reply:
formatting diff after reverting locale atom